lib/install/validate/validator_test_sim_util.go

```diff
@@ -209,6 +209,9 @@ func (v *Validator) validateForSim(ctx context.Context, input *data.Data) (*conf
 	vmRef := vm.Reference()
 	conf.SetMoref(&vmRef)
 
+	// Remove bridge network from main section of config, to better emulate real VC behaviour
+	conf.Networks["bridge"] = nil
```

Comment:
I'm not sure why this needs doing - what was the bug that you mentioned that relates to this?

Reply:
There are two changes related to this problem. The first is in rbac.go line 179. The original code was looking for the bridged network mo inside ExecutorConfig.Networks. That worked fine in the Simulator but failed when running against my VC test bed. For that to work I needed to change the rbac code to look at Network.ContainerNetworks. In case of the simulator the bridged network mo is found in both places. In case of my VC test bed it is only found in Network.ContainerNetworks. This modification tries to make the simulator behavior match the VC behavior, where ExecutorConfig.Networks can be nil.